### PR TITLE
MSI: Onboard RHODS scenario on OSD GCP

### PR DESCRIPTION
**Pull request by chetna14manku**
JIRA ticket: https://issues.redhat.com/browse/MSINTEROP-429

### CODE REVIEW

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml [hunk 1]
```diff
@@ -59,6 +59,36 @@ tests:
...
+      OPERATOR3_CONFIG: name=serverless-operator;namespace=openshift-serverless
+      OPERATOR4_CONFIG: name=openshift-pipelines-operator-rh;namespace=openshift-pipelines;channel=latest
+      OPERATOR5_CONFIG: name=nfd;namespace=openshift-nfd;target-namespaces=openshift-nfd,
+      PULL_SECRET_NAME: pull-secret
+      S3_BUCKET_NAME: openshift-cli-installer
+      S3_BUCKET_PATH: openshift-ci
+    test:
+    - ref: addon-tests-rhods
+    workflow: addons-and-operators
 - as: smoke-osd-rhods-addon-baseline-fips
```
Comment (chetna14manku) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml:92]: @rnetser @myakove now we'll have two OSD jobs, on AWS and OSD, so I would sugget to update the existing job name to reflect platform also, WDYT?
```suggestion
- as: smoke-osd-aws-rhods-addon-baseline-fips
```

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml (hunk 1 above)
Comment (omrirh) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml:92]: agreed; I think we'll need to specify the cloud provider on all existing OSD jobs and OCP/IPI to be onboarded as well

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml (hunk 1 above)
Comment (rnetser) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml:93]: @chetna14manku please update

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-gcp-rhods-addon-baseline-fips

**Review by rnetser — APPROVED** [review]

**Pull request merged**